### ISSUE TITLE
acknowledgement at the end of the specification

### ISSUE THREAD

**Issue by dboas** [fNIRS/snirf#9]
There have been many contributors.
I need to dig through the emails.
Mainly, it was started by Blaise Frederick and David Boas, with significant contributions from Ted, Jay, and Qianqian. And then there have been a community of suggestions that have been adopted.
Qianqian or Jay, can you update this in the spec?

**Comment by fangq** [comment]
if you attach a list of names/contact info, I will be happy to update, alternatively, you can click on the "Fork" button on the top-right, and make whatever edit in your fork, commit your change, and send a pull request to this repo. I can then merge the changes. It is quite easy to merge changes from multiple users on github.

**Comment by fangq** [comment]
no, I don't see it. click on the "Pull requests" in this repo, I only see my pull request on streaming support. 

have you create a request? you need to put fNIRS/snirf as base when creating the request.
